project/src/main/scala/com/mwz/sonar/scala/metadata/scalastyle/ScalastyleInspectionsGenerator.scala

```diff
@@ -50,10 +52,8 @@ object ScalastyleInspectionsGenerator {
     val templateFile = Paths
       .get(
         baseDirectory.value.toString,
-        "project",
-        "src",
-        "main",
-        "scala",
+        "project/src/main/scala",
+        "com/mwz/sonar/scala/metadata/scalastyle",
```

Comment:
Isn't the point of `Path.get` to do not use `/` because that won't work on **Windows**?

Reply:
ah yes, very good point - I'll fix that